demisto_sdk/__main__.py

```diff
@@ -911,40 +877,61 @@ def run_playbook(**kwargs):
     return playbook_runner.run_playbook()
 
 
-# ====================== json-to-outputs ====================== #
-@main.command('json-to-outputs')  # To no shadow json_to_outputs import
+# ====================== generate-outputs ====================== #
+@main.command()
 @click.help_option(
     '-h', '--help'
 )
 @click.option(
-    "-c", "--command", help="Command name (e.g. xdr-get-incidents)", required=True)
+    "-c", "--command", help="Specific command name (e.g. xdr-get-incidents)",
+    required=False)
 @click.option(
-    "-i", "--input",
+    "-j", "--json",
     help="Valid JSON file path. If not specified, the script will wait for user input in the terminal. "
          "The response can be obtained by running the command with `raw-response=true` argument.",
     required=False)
 @click.option(
-    "-p", "--prefix", help="Output prefix like Jira.Ticket, VirusTotal.IP, the base path for the outputs that the "
-                           "script generates", required=True)
+    "-p", "--prefix",
+    help="Output prefix like Jira.Ticket, VirusTotal.IP, the base path for the outputs that the "
+         "script generates", required=False)
 @click.option(
-    "-o", "--output", help="Output file path, if not specified then will print to stdout", required=False)
+    "-o", "--output",
+    help="Output file path, if not specified then will print to stdout",
+    required=False)
 @click.option(
-    "-v", "--verbose", is_flag=True, help="Verbose output - mainly for debugging purposes")
+    "-v", "--verbose", is_flag=True,
+    help="Verbose output - mainly for debugging purposes")
 @click.option(
-    "--interactive", help="If passed, then for each output field will ask user interactively to enter the "
-                          "description. By default is interactive mode is disabled", is_flag=True)
+    "--interactive",
+    help="If passed, then for each output field will ask user interactively to enter the "
+         "description. By default is interactive mode is disabled",
+    is_flag=True)
 @click.option(
     "-d", "--descriptions",
     help="A JSON or a path to a JSON file, mapping field names to their descriptions. "
          "If not specified, the script prompt the user to input the JSON content.",
     is_flag=True)
-def json_to_outputs_command(**kwargs):
+@click.option(
+    "-i", "--input",
+    help="Valid YAML integration file path.",
+    required=False)
```

Comment:
What do you think about removing -j flag and making input handle both JSON and YML?

Reply:
I'd like to distinguish between them, it seems more clear and reduces code.